wagtail/core/tests/test_streamfield.py

```diff
@@ -15,29 +16,34 @@
 from wagtail.images.tests.utils import get_test_image_file
 from wagtail.tests.testapp.models import (
     BlockCountsStreamModel,
+    JSONBlockCountsStreamModel,
+    JSONMinMaxCountStreamModel,
+    JSONStreamModel,
     MinMaxCountStreamModel,
     StreamModel,
 )
+from wagtail.utils.deprecation import RemovedInWagtail219Warning
 
 
 class TestLazyStreamField(TestCase):
+    model = StreamModel
+
     def setUp(self):
         self.image = Image.objects.create(
             title="Test image", file=get_test_image_file()
         )
-        self.with_image = StreamModel.objects.create(
+        self.with_image = self.model.objects.create(
             body=json.dumps(
                 [
                     {"type": "image", "value": self.image.pk},
                     {"type": "text", "value": "foo"},
                 ]
             )
         )
-        self.no_image = StreamModel.objects.create(
+        self.no_image = self.model.objects.create(
             body=json.dumps([{"type": "text", "value": "foo"}])
         )
-        self.nonjson_body = StreamModel.objects.create(body="<h1>hello world</h1>")
```

Comment:
This is not used in the test class at all and it causes problems with `JSONField`, so I removed it.